sdk/core/Azure.Core.Experimental/api/Azure.Core.Experimental.net461.cs

```diff
@@ -120,3 +120,73 @@ public ProvisionableTemplateAttribute(string resourceName) { }
         public string ResourceName { get { throw null; } }
     }
 }
+namespace System.ClientModel
```

Comment:
I don't think it should be in the root namespace. Should we put it in System.ClientModel.Auth?